### PR TITLE
Fix: try it logout

### LINKED ISSUE #465: Logout on Try it not working
**Describe the bug**
Logout on Try it is not working. 

**To Reproduce**
1. Go to https://review.docs.microsoft.com/en-us/graph/api/user-list?view=graph-rest-1.0&tabs=http&branch=master
2. Click on Try it
3. Login, then logout
4. Reload the page after logout
5. Surprise! You are logged in still.

**Expected behavior**
If you logout and reload the page, you should be logged out. 

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

### PR DESCRIPTION
**Pull request by thewahome**
## Overview 
Uses a workaround described in issue 1386  of [the msaljs repository](https://github.com/AzureAD/microsoft-authentication-library-for-js/).

Fixes #465 

### Demo

N/A 

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* Change the graphExplorerMode to TryIt
* Sign in using your preferred account
* Sign out and notice a pop up and choose the account you want to sign out from
* Reload the page
* You would need to sign out again :-)

### CODE REVIEW

File: src/app/services/graph-client/msal-service.ts [hunk 1]
```diff
@@ -72,6 +72,20 @@ export function logOut() {
   msalApplication.logout();
 }
 
+export function logOutPopUp() {
+  // @ts-ignore
```
Comment (jobala) [inline src/app/services/graph-client/msal-service.ts:76]: Why are we ignoring Typescript here

Comment (thewahome) [reply src/app/services/graph-client/msal-service.ts:76]: The methods I am accessing are private and surprisingly accessible as long as you ignore typescript.

It's an allowed workaround until the team releases a logoutPopUp implementation

**Comment by thewahome** [comment]
Good looking out. I have fixed it

File: src/app/services/graph-client/msal-service.ts (hunk 1 above)
Comment (jobala) [reply src/app/services/graph-client/msal-service.ts:76]: You can add this as a comment, so that we know to stop ignoring Typescript in the future.

**Pull request merged**